### ISSUE TITLE
Low-resolution Undocked Panels at 150% Display Scale

### ISSUE THREAD

**Issue by LcBeyer** [musescore/MuseScore#20681]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

When panels are undocked, they are rendered at a lower resolution than the rest of the application, making them appear blurry and unappealing.

### Steps to reproduce

1. Open MuseScore
2. Open a score
3. Drag a panel out of it's docked position and place it anywhere on the screen OR click the 3 dots and click "Undock"

### Screenshots/Screen recordings

<img width="960" alt="Screenshot 2023-12-23 171623" src="https://github.com/musescore/MuseScore/assets/154291622/77800e4b-bfb6-4e9c-b61b-8d653c2cb4be">

### MuseScore Version

MuseScore version (64-bit): 4.2.0-233521124, revision: github-musescore-musescore-eb8d33c

### Regression

I don't know

### Operating system

Windows 11 Home

### Additional context

This bug does not occur on my dad's Windows 10 PC on the same version of MuseScore. If it helps, my laptop uses the integrated Radeon graphics of its AMD Ryzen 5 5500U, driver version 22.20.12-220707a-381421C-Lenovo, and his PC has a GTX 1080 with GeForce Game Ready Driver version 537.58.

**Comment by LcBeyer** [comment]
I plugged a second monitor into the laptop that was having the issue, and I discovered that the problem seems to be related to the monitor's display scale.

My laptop's integrated screen is set to 150% scaling by default, and all of the other displays that didn't have the issue were set to 100%. To see if this was the cause of the issue, I set my laptop's integrated display to 100% as well, and undocked panels were rendered properly, which makes it seem likely.

**Comment by zacjansheski** [comment]
Not a very helpful video, difficult to get good quality with 2 monitors.  But I have one monitor set to 150%.  Dragging panels between monitors makes the content blurry temporarily, corrects on hovering/re-sizing

https://github.com/user-attachments/assets/61faa9d1-8272-47c9-80a3-934b87d177bd

**Comment by LcBeyer** [comment]
> Dragging panels between monitors makes the content blurry temporarily, corrects on hovering/re-sizing

Yes, that is now the case for me as well. It seems to me that one of the many updates in the past two years partially resolved the issue, though I don't recall when I last experienced it as my post describes.
